src/funding/deposit-grid/deposit-grid.scss

```diff
@@ -0,0 +1,103 @@
+@import "styles/styles.scss";
+@import "src/resources/elements/primeDesignSystem/styles";
+@import "src/resources/elements/primeDesignSystem/variables";
+@import "src/resources/elements/primeDesignSystem/colors";
+.deposit {
```

Comment:
I don't see this class being used by deposit-grid.  I would recommend calling it "deposit-grid" if it is the container of the component by that name.